modules/Bio/EnsEMBL/Production/Pipeline/ProteinFeatures/InterProScanVersionCheck.pm

```diff
@@ -54,9 +65,29 @@ sub run {
         $self->throw("Could not find version in service file:\n$service_version_file\n$data");
       }
     }
+
+    # To guard against the 'current' version changing during pipeline
```

Comment:
I don't see it from the code (surely my fault), but is the process retrieving "where" the symlink is pointing to and register it? Or is it done another way?

Reply:
No, that would probably have been a cleverer way to do it. Instead, it works on the assumption that the subdir for a specific version has the format `interproscan-<version>`, and since we have just saved the version into a parameter, we just concatentate things together to get the path.